### ISSUE TITLE
Bug report: TorrentPotato implementation not returning  "imdb_id" in results.

### ISSUE THREAD

**Issue by qja** [cardigann/cardigann#181]
### Issue experienced
curl "http://192.168.6.88:5060/torrentpotato/torrentleech?imdbid=tt0387808&search=Idiocracy&passkey=d51884b7e8845734cfae0bf054dfbaa3"
{
  "results": [
    {
      "release_name": "Idiocracy 2006 720p HDTV x264-x0r",
      "torrent_id": "https://www.torrentleech.org/torrent/538891",
      "details_url": "https://www.torrentleech.org/torrent/538891#comments",
      "download_url": "http://192.168.6.88:5060/download/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJsIjoiaHR0cHM6Ly93d3cudG9ycmVudGxlZWNoLm9yZy9kb3dubG9hZC81Mzg4OTEvSWRpb2NyYWN5LjIwMDYuNzIwcC5IRFRWLngyNjQteDByLnRvcnJlbnQiLCJuYmYiOjE0NzgwMjg0NzYsInMiOiJ0b3JyZW50bGVlY2gifQ.UEhsL2EyS_YpIszU3kftj2uiw5I9StoUdh8VJDkkCWc/Idiocracy+2006+720p+HDTV+x264-x0r.torrent",
      "imdb_id": "",
      "freeleech": false,
      "type": "movie",
      "size": 1337890,
      "leechers": 1,
      "seeders": 136
    }
  ],
  "total_results": 1
### Steps to reproduce
Install cardigan and at least on torrent leech result provided by cardigann have empty "imdb_id". 
From a quick browse through the code it seems to never be set from torznab Result object so I suppose it's the same issue for all providers. Probably should be set from the query .

This field seems important:
From https://github.com/CouchPotato/CouchPotatoServer/wiki/CouchPotato-Torrent-Provider 
imdb_id: string id of imdb url starting with "tt" (mandatory for Movies, optional for TV series)

### Cardigann version (in the footer, or via `cardigann --version`)

1.9.3

Regards,
Q

**Comment by lxt2** [comment]
Thanks, any idea if this causes issues with Couchpotato? Seems like it should be an easy fix.

**Comment by qja** [comment]
I don't think this causes an issue per se but I guess if the spec asks for it if it's not too much trouble it should be implemented.
